0.download_data/jump_dataset.csv

```diff
@@ -0,0 +1,51 @@
+source_4,2020_11_04_CPJUMP1,BR00116991,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00116991/BR00116991.sqlite,COMPOUND
```

Comment:
I have three additional comments on this file.

1. The name is not very specific. Is there something you can think of that would be more accurate? Maybe `jump_dataset_location_manifest.csv`?
2. When you're creating these files, it is super important to have column names. This will tell everyone (including future you!) what each of these columns represent
3. Please place this file inside a folder (maybe `data`?) this will help to organize the repo and make it more readable